src/components/manifold-oauth/manifold-oauth.tsx

```diff
@@ -4,11 +4,11 @@ import { Event, EventEmitter, Component, h, Prop } from "@stencil/core";
   tag: "manifold-oauth"
 })
 export class ManifoldOauth {
-  @Prop() oauthUrl: string;
-  @Event() tokenReceived: EventEmitter;
+  @Prop() oauthUrl?: string = "https://login.manifold.co/signin/oauth/web";
```

Comment:
Does making it optional work because there's a default?

Reply:
correct :)